src/main/java/uk/gov/hmcts/reform/hmc/model/CaseDetails.java

```diff
@@ -27,9 +26,6 @@
     @Pattern(regexp = "^\\d{16}$", message = ValidationError.CASE_REF_INVALID)
     private String caseRef;
 
-    @NotNull(message = ValidationError.REQUEST_TIMESTAMP_EMPTY)
-    private LocalDateTime requestTimeStamp;
```

Comment:
Been removed since we also have a field named `hearingRequestReceivedDateTime` and as per latest "Hearing Principles" we populate it in code rather than asking the consumer to provide us with a time they've sent it!